src/omv/sensors/paj6100.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * This file is part of the OpenMV project.
+ * Copyright (c) 2019 Lake Fu <lake_fu@pixart.com>
+ * This work is licensed under the MIT license, see the file LICENSE for details.
+ *
+ * PAJ6100 driver.
+ *
+ */
+#ifndef __PAJ6100_H__
+#define __PAJ6100_H__
+#if (OMV_ENABLE_PAJ6100 == 1)
```

Comment:
```suggestion
```
@Lak4CYUT This will cause a build error

Reply:
It is cause by https://github.com/openmv/openmv/pull/1365#discussion_r652977159.
I don't see that you remove the #endif.

Reply:
Yes sorry it was supposed to be remove too.